src/components/FolderCard.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import elementStyles from '../styles/isomer-cms/Elements.module.scss';
+import contentStyles from '../styles/isomer-cms/pages/Content.module.scss';
+
+const FolderCard = ({
+  displayText,
+  link,
+  settingsToggle,
+  itemIndex
+}) => (
+  <div className={contentStyles.folderContainerBoxes}>
+    <div className={contentStyles.boxesContainer}>
+      <Link className={`${contentStyles.component} ${contentStyles.card} ${elementStyles.card}`} to={link}>
+        <i className="bx bxs-folder" />
```

Comment:
Can we make the icon slightly bigger? It's a little small right now

<img width="460" alt="Screenshot 2020-11-04 at 4 45 59 AM" src="https://user-images.githubusercontent.com/31984694/98038997-f87f9c00-1e58-11eb-9ea3-40a67685964c.png">

For reference, here was the design on Figma:
<img width="636" alt="Screenshot 2020-11-04 at 4 48 01 AM" src="https://user-images.githubusercontent.com/31984694/98039021-02090400-1e59-11eb-8530-80fb4c025fca.png">

Reply:
Resolved in https://github.com/isomerpages/isomercms-frontend/pull/184/commits/ff457eba141c45610f88b60f2e76d5d25c86346d